api/views/personal_info.py

```diff
@@ -76,3 +82,20 @@ def check_existence_of_chat_id(self, request: Request) -> Response:
         serializer = self.get_serializer(data=request.query_params)
         serializer.is_valid(raise_exception=True)
         return Response(request.query_params)
+
+    @extend_schema(
+        responses={
+            status.HTTP_200_OK: OpenApiResponse(
+                response=PersonalInfoSerializer,
+                description="Users with these parameters exist",
+            ),
+            status.HTTP_406_NOT_ACCEPTABLE: OpenApiResponse(
```

Comment:
I suggest to move it to custom schema, because now every GET handler could return 406.

Reply:
not relevant anymore: returning 200 OK and empty list